andela/Ah-frontend-guardians
# #162685632 Users can like and or dislike an article

#### What does this PR do?
This PR enables only authenticate users to like and unlike an article whether or not it was authored by them.

#### Description of Task to be completed?
* Like and unlike article functionality
* Dislike and Undislike Article functipnality
* UI for like and dislike

#### How should this be manually tested?
```Use the appropriate screenshots to test the expected results```
* Visit the URL below:
https://ah-frontend-guardians-pr-20.herokuapp.com/

* Click on an article
* Then you can click on the like and or dislike buttons

#### What are the relevant pivotal tracker stories?
[#162685632](https://www.pivotaltracker.com/story/show/162685632)

#### Screenshots 
![image](https://user-images.githubusercontent.com/26769886/53621693-61d0f580-3c08-11e9-8f62-c7f903570527.png)

![image](https://user-images.githubusercontent.com/26769886/53621696-65647c80-3c08-11e9-8618-7ddd1549a577.png)
